app/containers/V2FarmingPage/index.js

```diff
@@ -474,32 +522,32 @@ export function FarmingPage(props) {
 
   const poolInForAlloc = async () => {
     if (wallet.signer !== 'signer') {
-      const masterChefV2 = await masterChefV2Contract();
-      const rigelAllocPoint = await masterChefV2.poolInfo(0);
+      const masterChef = await masterChefContract();
+      const rigelAllocPoint = await masterChef.poolInfo(0);
```

Comment:
Also this, I don't think we are suppose to have anything like masterchef again, but masterchefV2 and masterchefV2Contract, you can confirm this from Kay